src/certificate.ts

```diff
@@ -105,6 +122,7 @@ export default class Certificate {
 
     // Set locale
     this.locale = domain.i18n.ensureIsSupported(this.options.locale === 'auto' ? domain.i18n.detectLocale() : this.options.locale);
+    this.explorerAPIs = this.options.explorerAPIs || [];
```

Comment:
what if we were handling the whole list from here, as default options. That way the reconciliation happens early